### PR TITLE
[CLOUDCRAFT-716] Update Cloudcraft's "Cancel your Cloudcraft Subscription" article

### PR DESCRIPTION
**Pull request by jamesponddotco**
### What does this PR do? What is the motivation?
Make the article clearer, explain things with more details, and follow the style guide more closely.

### Merge instructions
- [x] Please merge after reviewing

### Additional notes
I'm not sure if the title follows the style guide. Maybe it should be "Cancel your Cloudcraft subscription" instead?

### CODE REVIEW

File: content/en/cloudcraft/account-management/cancel-subscription.md [hunk 1]
```diff
@@ -3,17 +3,32 @@ title: Cancel your Cloudcraft Subscription
 kind: documentation
 ---
 
-We're sorry to see you go! Only the Account Owner can cancel a Cloudcraft subscription, which takes effect at the end of the current billing cycle. When you cancel your subscription, your account is downgraded to the free plan. 
+Canceling your Cloudcraft subscription is super easy and can be done in a few clicks from inside the application.
```
Comment (rtrieu) [inline content/en/cloudcraft/account-management/cancel-subscription.md:6]:
```suggestion
Canceling your Cloudcraft subscription can be done from inside the Cloudcraft application.
```

File: content/en/cloudcraft/account-management/cancel-subscription.md [hunk 2]
```diff
@@ -3,17 +3,32 @@ title: Cancel your Cloudcraft Subscription
...
 
-You won't lose access to the diagrams you created. However, **Infinite Grid** blueprints will switch to read-only mode until you subscribe again or change them to the standard grid size, and AWS resources will no longer update.
+Only the account owner can cancel a Cloudcraft subscription, which takes effect at the end of your current billing cycle. When you cancel your subscription, your account is downgraded to the free plan.
 
-<div class="alert alert-info">If you're canceling within our 30-day money-back guarantee period and prefer a refund, <a href="https://app.cloudcraft.co/support" rel="help" title="Cloudcraft support">reach out to our support team</a> instead.
+You won't lose access to the diagrams you created, but you'll lose access to a few features.
```
Comment (rtrieu) [inline content/en/cloudcraft/account-management/cancel-subscription.md:10]:
```suggestion
Upon cancellation, you will still have access to the diagrams you created, but will lose access to a few features:
```

File: content/en/cloudcraft/account-management/cancel-subscription.md [hunk 3]
```diff
@@ -3,17 +3,32 @@ title: Cancel your Cloudcraft Subscription
...
+
+- _Infinite grid_ blueprints will switch to read-only mode.
```
Comment (rtrieu) [inline content/en/cloudcraft/account-management/cancel-subscription.md:12]:
```suggestion
- _Infinite grid_ blueprints switch to read-only mode.
```

File: content/en/cloudcraft/account-management/cancel-subscription.md [hunk 4]
```diff
@@ -3,17 +3,32 @@ title: Cancel your Cloudcraft Subscription
...
+- The _Live_ feature will no longer be available.
```
Comment (rtrieu) [inline content/en/cloudcraft/account-management/cancel-subscription.md:13]:
```suggestion
- The _Live_ feature is no longer be available.
```

File: content/en/cloudcraft/account-management/cancel-subscription.md [hunk 5]
```diff
@@ -3,17 +3,32 @@ title: Cancel your Cloudcraft Subscription
...
+- AWS and Azure resources will no longer update.
```
Comment (rtrieu) [inline content/en/cloudcraft/account-management/cancel-subscription.md:14]:
```suggestion
- AWS and Azure resources no longer get updated.
```

File: content/en/cloudcraft/account-management/cancel-subscription.md [hunk 6]
```diff
@@ -3,17 +3,32 @@ title: Cancel your Cloudcraft Subscription
...
+- [The Bits menu][1] will stop working.
```
Comment (rtrieu) [inline content/en/cloudcraft/account-management/cancel-subscription.md:15]:
```suggestion
- [The Bits menu][1] no longer works.
```

File: content/en/cloudcraft/account-management/cancel-subscription.md [hunk 7]
```diff
@@ -3,17 +3,32 @@ title: Cancel your Cloudcraft Subscription
...
 1. Navigate to **User** > **Subscription Settings**.
-2. Click **Cancel subscription**.
+2. Select **Cancel subscription**.
 3. Optionally, provide feedback on what Cloudcraft can do to be a better fit for you in the future.
-4. Click **Cancel my account**.
+4. Select **Cancel my account**.
+
+If you do not see the **Cancel subscription** option, you may have bought your subscription through our sales team or through the AWS Marketplace.
+
+- If you bought your subscription through our sales team, it won't auto renew and will expire at the end of your billing cycle.
```
Comment (rtrieu) [inline content/en/cloudcraft/account-management/cancel-subscription.md:27]:
```suggestion
- If you bought your subscription through our sales team, it does not auto-renew and expires at the end of your billing cycle.
```

File: content/en/cloudcraft/account-management/cancel-subscription.md [hunk 8]
```diff
@@ -3,17 +3,32 @@ title: Cancel your Cloudcraft Subscription
...
+- If you bought your subscription through the AWS Marketplace, the cancellation process must be done by you through your AWS account. For more information, see [Cancel your product subscription][2] in the AWS Marketplace documentation page.
```
Comment (rtrieu) [inline content/en/cloudcraft/account-management/cancel-subscription.md:28]:
```suggestion
- If you bought your subscription through the AWS Marketplace, the cancellation process must be done through your AWS account. For more information, see [Cancel your product subscription][2] in the AWS Marketplace documentation page.
```

File: content/en/cloudcraft/account-management/cancel-subscription.md (hunk 1 above)
Comment (rtrieu) [reply content/en/cloudcraft/account-management/cancel-subscription.md:6]: Can you add the exact instructions on how to cancel the subscription? :)

**Review by rtrieu — CHANGES_REQUESTED** [review]
Some minor feedback, but otherwise looks good!

File: content/en/cloudcraft/account-management/cancel-subscription.md (hunk 1 above)
Comment (jamesponddotco) [reply content/en/cloudcraft/account-management/cancel-subscription.md:6]: They should be a little farther down, I believe.

**Comment by jamesponddotco** [comment]
@rtrieu All suggestions have been applied :)

File: content/en/cloudcraft/account-management/cancel-subscription.md [hunk 9]
```diff
@@ -3,17 +3,32 @@ title: Cancel your Cloudcraft Subscription
...
 </div>
 
 1. Navigate to **User** > **Subscription Settings**.
-2. Click **Cancel subscription**.
+2. Select **Cancel subscription**.
 3. Optionally, provide feedback on what Cloudcraft can do to be a better fit for you in the future.
-4. Click **Cancel my account**.
+4. Select **Cancel my account**.
+
+If you do not see the **Cancel subscription** option, you may have bought your subscription through our sales team or through the AWS Marketplace.
```
Comment (rtrieu) [inline content/en/cloudcraft/account-management/cancel-subscription.md:25]:
```suggestion
If you do not see the **Cancel subscription** option, you may have bought your subscription through the Datadog sales team or through the AWS Marketplace.
```

File: content/en/cloudcraft/account-management/cancel-subscription.md [hunk 10]
```diff
@@ -3,17 +3,32 @@ title: Cancel your Cloudcraft Subscription
...
+
+- If you bought your subscription through our sales team, it does not auto-renew and expires at the end of your billing cycle.
```
Comment (rtrieu) [inline content/en/cloudcraft/account-management/cancel-subscription.md:27]:
```suggestion
- If you bought your subscription through the Datadog sales team, it does not auto-renew and expires at the end of your billing cycle.
```

File: content/en/cloudcraft/account-management/cancel-subscription.md [hunk 11]
```diff
@@ -3,17 +3,32 @@ title: Cancel your Cloudcraft Subscription
...
 ---
 
-We're sorry to see you go! Only the Account Owner can cancel a Cloudcraft subscription, which takes effect at the end of the current billing cycle. When you cancel your subscription, your account is downgraded to the free plan. 
+Canceling your Cloudcraft subscription can be done from inside the Cloudcraft application.
 
-You won't lose access to the diagrams you created. However, **Infinite Grid** blueprints will switch to read-only mode until you subscribe again or change them to the standard grid size, and AWS resources will no longer update.
+Only the account owner can cancel a Cloudcraft subscription, which takes effect at the end of your current billing cycle. When you cancel your subscription, your account is downgraded to the free plan.
 
-<div class="alert alert-info">If you're canceling within our 30-day money-back guarantee period and prefer a refund, <a href="https://app.cloudcraft.co/support" rel="help" title="Cloudcraft support">reach out to our support team</a> instead.
+Upon cancellation, you will still have access to the diagrams you created, but will lose access to a few features:
```
Comment (rtrieu) [inline content/en/cloudcraft/account-management/cancel-subscription.md:10]:
```suggestion
Upon cancellation, you can still access to the diagrams you created, but will lose access to a few features:
```

**Review by rtrieu — COMMENTED** [review]
A few more minor ones!

**Comment by jamesponddotco** [comment]
@rtrieu Done. Sorry for the delay :S

**Review by rtrieu — APPROVED** [review]
Looks great, thank you!

**Pull request merged**